### PR TITLE
Added new script for download youtube videos directly using script

### PR DESCRIPTION
**Pull request by rahulbandari2499**

### CODE REVIEW

File: Youtube-Downloader/down.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+			c+=1
+		if flag==1:
+			index=raw_input("Enter index")
+			index=int(index)
+		elif qual==2:
+			index=c-1
+		elif qual==1:
+			index=0
+		filename=video['items'][i]['pafy'].streams[index]
+		x=filename.download(filepath="D:/python")
```
Comment (amitsagtani97) [inline Youtube-Downloader/down.py:50]: filepath should not be hard coded as "D:/python" path is not always available(e.g in linux os). Instead take input from user, where he/she wants to store the downloaded video, or download it in the directory containing this script.

Comment (rahulbandari2499) [reply Youtube-Downloader/down.py:50]: OK i will be updating the code soon!!!

Comment (DanielXYZ2000) [reply Youtube-Downloader/down.py:50]: I'm waiting for the merge into the master branch - that I can try the code :-)
By the way I would not use type as a variable name - maybe download_mode could be a better variable name

**Review by amitsagtani97 — APPROVED** [review]
Thanks for the PR :)

**Pull request merged**